src/compiler/crystal/command/spec.cr

```diff
@@ -67,7 +67,8 @@ class Crystal::Command
     source_filename = File.expand_path("spec")
 
     source = target_filenames.map { |filename|
-      %(require "./#{::Path[filename].to_posix}")
+      file_path = ::Path[filename].to_posix
+      %(require "#{file_path.absolute? ? file_path : "./#{file_path}"})
```

Comment:
Readability-related suggestion:

```suggestion
      file_path = "./#{file_path}" unless file_path.absolute?
      %(require "#{file_path}"})
```

Reply:
@Sija we still need to call `.to_posix`, right?

Reply:
@KevinSjoberg yup (already done at line no. 70).

Reply:
Ah, my bad. Was a bit confused by suggestion dialog. Looks good. 🙂